### ISSUE TITLE
Please, update dependencies

### ISSUE THREAD

**Issue by osmirnova-voc** [pushy/pushy-flutter#64]
Because no versions of pushy_flutter match >2.0.18 <3.0.0 and pushy_flutter 2.0.18 depends on js ^0.6.5, pushy_flutter ^2.0.18 requires js ^0.6.5.
And because flutter_sound >=9.4.6 depends on flutter_sound_web 9.4.6 which depends on js ^0.7.1, pushy_flutter ^2.0.18 is incompatible with flutter_sound >=9.4.6.

**Comment by pushy** [comment]
Hi @osmirnova-voc,
Thanks for reporting this issue. However, we are unable to reproduce it on a brand new test project.

Here are the steps we followed:

1. Run the flutter create command.

```
flutter create test_drive
```

2. Change to the Flutter project directory.

```
cd test_drive
```

3. Edit the `pubspec.yaml` to include the following under the `dependencies` section:

```
pushy_flutter: 2.0.18

```

4. The SDK installed successfully after running the following command:

```
flutter pub get
```

Please give the above steps a try and let us know if you still face an issue.

**Comment by pushy** [comment]
Hi @osmirnova-voc,
Sure thing! We've updated the `js` package to version `0.7.1` as requested in 472a5be and published version `2.0.19`.

Install version `2.0.19` of our [Flutter SDK](https://pub.dev/packages/pushy_flutter) by editing the `pubspec.yaml` in the root directory of your project and specifying the following under the dependencies section:

```
pushy_flutter: 2.0.19
```

Run `flutter pub get` to fetch the dependency.

Please let us know if this resolves your issue, or if you require any further assistance.

**Comment by osmirnova-voc** [comment]
Thank you very much for your professionalism and swift service. It's truly a delight to use your service!
